components/admin_console/admin_navbar_dropdown/admin_navbar_dropdown.tsx

```diff
@@ -1,35 +1,35 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import PropTypes from 'prop-types';
 import React from 'react';
-import {FormattedMessage, injectIntl} from 'react-intl';
+import {FormattedMessage, injectIntl, IntlShape} from 'react-intl';
+
+import {Team} from 'mattermost-redux/types/teams';
 
 import * as GlobalActions from 'actions/global_actions.jsx';
 
 import {filterAndSortTeamsByDisplayName} from 'utils/team_utils.jsx';
 import {ModalIdentifiers} from 'utils/constants';
-import {intlShape} from 'utils/react_intl';
 
 import AboutBuildModal from 'components/about_build_modal';
 
 import Menu from 'components/widgets/menu/menu';
 
 import MenuItemBlockableLink from './menu_item_blockable_link';
 
-class AdminNavbarDropdown extends React.Component {
-    static propTypes = {
-        intl: intlShape.isRequired,
-        locale: PropTypes.string.isRequired,
-        siteName: PropTypes.string,
-        navigationBlocked: PropTypes.bool,
-        teams: PropTypes.arrayOf(PropTypes.object).isRequired,
-        actions: PropTypes.shape({
-            deferNavigation: PropTypes.func,
-        }).isRequired,
-    }
+interface AdminNavbarDropdownProps {
```

Comment:
We are using everywhere:
```js
type Props = {
   ...
}
```

I think is cleaner and more consistent with the rest of the code.